BoxContentSDK/BoxContentSDK/Clients/BOXContentClient_Private.h

```diff
@@ -20,6 +20,7 @@
 @property (nonatomic, readwrite, strong) BOXAppUserSession *appSession;
 @property (nonatomic, readwrite, strong) BOXAbstractSession *session;
 @property (nonatomic, readonly, strong) BOXURLSessionManager *urlSessionManager;
+@property (nonatomic, readwrite, copy) NSString *rootCacheDir;
```

Comment:
we should probably call it tempCacheDir to differentiate with the rootCacheDir passed in the oneTimeSetUp* methods.